### PR TITLE
Update strategy.md

### PR DESCRIPTION
**Pull request by arnuojo**
Translation version of strategy.md

### CODE REVIEW

File: content/zh-CN/guides/strategy.md [hunk 1]
```diff
@@ -1,270 +1,288 @@
...
+### 启动成功的开源项目
+
+[Linux基金会：启动成功的开源项目](http://event.lvl3.on24.com/event/14/07/58/6/rt/1/documents/resourceList1493819210084/how_to_start_a_successful_open_source_project.pdf)
+
+### 法务资源
+
+* [国际自由和开源软件法律评论](http://www.ifosslr.org/ifosslr)
+* [开源合规计划](https://compliance.linuxfoundation.org/)
+
+### Licensing开源许可证
```
Comment (node) [inline content/zh-CN/guides/strategy.md:257]: 英文可以不要？

**Review by node — CHANGES_REQUESTED** [review]

File: content/zh-CN/guides/strategy.md [hunk 2]
```diff
@@ -1,270 +1,288 @@
...
+让我们先从最终目标开始，然后解释如何实现目标。你的目标是在一个文档中详细说明你的开源战略，从而能让不同角色的每个人（从研发、法务，到营销和管理高层CXO）都可以参考。本战略文档将帮助您的团队了解您的开源项目背后的业务目标，确保更好的决策，并将风险降至最低。
 
-> "At Salesforce, we have internal documents that we circulate to our engineering team, providing strategic guidance and encouragement around open source. These encourage the creation and use of open source, letting them know in no uncertain terms that the strategic leaders at the company are fully behind it. Additionally, if there are certain kinds of licenses we don't want engineers using, or other open source guidelines for them, our internal documents need to be explicit."* – [Ian Varley](https://www.linkedin.com/in/ianvarley/), Software Architect at Salesforce.
+>“在Salesforce，我们向我们的研发团队分发内部文档，提供关于开源的战略指导和导向。这些导向鼓励开源的创建和使用，让他们毫不含糊地知道，且公司的战略领导者完全支持它。此外，如果我们不希望研发人员使用某些类型的开源许可证，或有其它的一些开源指南，都需要在我们的内部文档中明确。” —— [Ian Varley](https://www.linkedin.com/in/ianvarley/),  Salesforce的软件架构师。
```
[the remark this answers is not in the document]
Comment (arnuojo) [reply content/zh-CN/guides/strategy.md:31]: +1

File: content/zh-CN/guides/strategy.md [hunk 3]
```diff
@@ -1,270 +1,288 @@
...
+在这一点上，创建一个开源项目是无价的。TODO Group的开源指南 [《创建一个开源项目》](https://www.linuxfoundation.org/resources/open-source-guides/creating-an-open-source-program/)  指出："通过创建开源项目办公室OSPO，企业可以是能、简化和组织开源的使用，并使其直接与公司的长期业务计划挂钩。开源项目办公室旨在成为公司开源运营和组织架构的中心，帮助将所有需要的组件凝聚在一起。
```
[the remark this answers is not in the document]
Comment (arnuojo) [reply content/zh-CN/guides/strategy.md:98]: 之前是错别字，应该是“”使能“

**Review by arnuojo — COMMENTED** [review]
[strategy _2024.6.5.md](https://github.com/user-attachments/files/15581600/strategy._2024.6.5.md)
modified based on node's comments

**Review by node — APPROVED** [review]
LGTM

**Comment by node** [comment]
Please change your PR's draft status so that others can merge it.

**Pull request merged**